src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -695,9 +695,14 @@ def defineFaceAnnotations(cls, region, options, annotationGroups):
         :param annotationGroups: List of annotation groups for top-level elements.
         New face annotation groups are appended to this list.
         """
-        try:
-            # create fissure groups
-            fm = region.getFieldmodule()
+        parameterSetName = options['Base parameter set']
+        isMouse = 'Mouse 1' in parameterSetName
+        isHuman = 'Human 1' in parameterSetName
+
+        # create fissure groups
+        fm = region.getFieldmodule()
+
+        if isHuman:
```

Comment:
I suggest splitting up the code into left and right; left for human only, right can be run for human and mouse.